engine/runtime/src/main/java/org/enso/interpreter/node/callable/resolver/HostMethodCallNode.java

```diff
@@ -59,7 +79,13 @@ public enum PolyglotCallType {
      * @return true if one can directly pass this object to {@link InteropLibrary}
      */
     public boolean isInteropLibrary() {
-      return this != NOT_SUPPORTED && this != CONVERT_TO_TEXT && this != CONVERT_TO_DATE;
+      return this != NOT_SUPPORTED
```

Comment:
The list is getting longer. A time will may have arrived to give the enum `final boolean interopLibrary` field and constructor.